omero/sysadmins/cli/light-admins.rst

```diff
@@ -65,6 +65,8 @@ removes a restriction so that they may exercise `SomePrivilege`.
   You may not clear a restriction from an administrator if you have that
   same restriction applying to yourself.
 
+.. _ReadSession-warning:
```

Comment:
not sure why is it here ? This is not visible in the built version. The other warnings are called just "warning", even here in the doc ?

Reply:
aha, got it, you need this anchor to be able to call the warning from the https://github.com/openmicroscopy/ome-documentation/pull/1930/files#diff-0ff5c8ce87cff45a62568c35b1f915d8